app/assets/javascripts/admin/mixins/setting-component.js.es6

```diff
@@ -100,6 +100,24 @@ export default Mixin.create({
     return settingDefault !== bufferedValue;
   },
 
+  @discourseComputed("buffered.value")
+  bufferedValues(bufferedValuesString) {
+    return bufferedValuesString && bufferedValuesString.split("|");
```

Comment:
can `bufferedValuesString`  be a `""` ? If yes we should filter `.filter(Boolean)`  as `"".split("|") // [""]`